src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -38,4 +38,9 @@ public class Message {
     private OffsetDateTime creationTimestamp;
     @UpdateTimestamp
     private OffsetDateTime updateTimestamp;
+    @Builder.Default
+    String templateName = "default";
+    @Transient
+    @JsonDeserialize
+    Map<String, Object> templateParams;
```

Comment:
Do we need it in `Message` if it is not going to be persisted?

Reply:
In order to leave the door open for a different body for sms & email, I thought it would be best to have the template info at the message level, and remove the body & subject once templates are implemented for email.

Reply:
I see what you meant now, I changed it